<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>589</width>
    <height>622</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="maximumSize">
   <size>
    <width>589</width>
    <height>622</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Audio Stream</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="4" column="0">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Data:</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QComboBox" name="comboBox_liveFileStream">
      <item>
       <property name="text">
        <string>Choose Stream Type</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Live Stream</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>File Stream</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="4" column="1" colspan="3">
     <widget class="QTextBrowser" name="textBrowser"/>
    </item>
    <item row="3" column="1">
     <widget class="QPushButton" name="pushButton_chooseFile">
      <property name="text">
       <string>Choose File</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QComboBox" name="comboBox_senderReceiver">
      <item>
       <property name="text">
        <string>Choose Type</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Sender</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Receiver</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="3" column="2">
     <widget class="QPushButton" name="pushButton_start">
      <property name="text">
       <string>Start</string>
      </property>
      <property name="checkable">
       <bool>true</bool>
      </property>
      <property name="checked">
       <bool>false</bool>
      </property>
      <property name="autoRepeat">
       <bool>false</bool>
      </property>
      <property name="autoExclusive">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item row="3" column="3">
     <widget class="QPushButton" name="pushButton_pause">
      <property name="text">
       <string>Pause</string>
      </property>
      <property name="checkable">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="2" column="2" colspan="2">
     <widget class="QLineEdit" name="lineEdit_IP">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QComboBox" name="comboBox_cast">
      <item>
       <property name="text">
        <string>Choose Cast</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Unicast</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Multicast</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="label_Icon_RS">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="maximumSize">
       <size>
        <width>40</width>
        <height>40</height>
       </size>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap resource="resource.qrc">:/Images/RS_usa.jpeg</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>589</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuTools">
    <property name="title">
     <string>Tools</string>
    </property>
    <addaction name="actionOptions"/>
   </widget>
   <addaction name="menuTools"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionOptions"/>
  </widget>
  <action name="actionOptions">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/Images/settingsIcon.png</normaloff>:/Images/settingsIcon.png</iconset>
   </property>
   <property name="text">
    <string>Options</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="resource.qrc"/>
 </resources>
 <connections/>
</ui>
